pkg/engine/engine_test.go

```diff
@@ -622,3 +623,65 @@ func TestLikelyDuplicate(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkPopulateMatchingDetectors(b *testing.B) {
+	allDetectors := DefaultDetectors()
+	ac := ahocorasick.NewAhoCorasickCore(allDetectors)
+
+	// Generate sample data with keywords from detectors.
+	dataSize := 1 << 20 // 1 MB
+	sampleData := generateRandomDataWithKeywords(dataSize, allDetectors)
+
+	smallChunk := 1 << 10  // 1 KB
+	mediumChunk := 1 << 12 // 4 KB
+	largeChunk := 1 << 14  // 16 KB
+	xlChunk := 1 << 15     // 32 KB
+	xxlChunk := 1 << 16    // 64 KB
+	xxxlChunk := 1 << 18   // 256 KB
+	chunkSizes := []int{smallChunk, mediumChunk, largeChunk, xlChunk, xxlChunk, xxxlChunk}
+
+	for _, chunkSize := range chunkSizes {
+		b.Run(fmt.Sprintf("ChunkSize_%d", chunkSize), func(b *testing.B) {
+			b.ReportAllocs()
+			b.SetBytes(int64(dataSize))
```

Comment:
Update this to chunkSize to update the throughput on the benchmarks

Reply:
I made a mistake; the screenshot above was from a detector benchmark in `aws_test.go`. This benchmark should compare the performance of `FindDetectorMatch` across different chunk sizes. I intended to show that using a larger chunk size (64kB) can increase throughput without significantly affecting the operation's duration. This was my basis for suggesting an increase in the default chunk size from 10kB to 64kB.

Here is the correct benchmark:
![Screenshot 2024-05-16 at 2 49 06 PM](https://github.com/trufflesecurity/trufflehog/assets/21311841/328b51f2-285e-43f0-b189-500f81c0c438)
:

Reply:
I made a mistake; the screenshot above was from a detector benchmark in `aws_test.go`. This benchmark should compare the performance of `FindDetectorMatch` across different chunk sizes. 

Here is the correct benchmark:
![Screenshot 2024-05-16 at 3 40 02 PM](https://github.com/trufflesecurity/trufflehog/assets/21311841/58a14cc5-5606-455e-b2e9-55388732ae0a)